packages/core/src/subscriptions/client/deleteSubscription.js

```diff
@@ -0,0 +1,26 @@
+import client, { adaptError } from '../../helpers/client';
+import join from 'proper-url-join';
+
+/**
+ * Method responsible for sending a delete all subscriptions request to MKT API containing the subscription id and email hash of the user to be unsubscribed.
+ *
+ * @function deleteSubscriptions
+ * @memberof module:subscriptions/client
+ *
+ * @param {object} props - Props object.
```

Comment:
I would call this params instead of props as this is not a React Component.

Reply:
Updated. Can you check again?